src/SonarScanner.MSBuild.Shim/SonarScanner.Wrapper.cs

```diff
@@ -54,7 +54,7 @@ public class SonarScannerWrapper : ISonarScanner
 
         private const string CmdLineArgPrefix = "-D";
 
-        private const string SonarScannerVersion = "3.0.3.778";
+        private const string SonarScannerVersion = "3.1.0.1141";
```

Comment:
Please add a document on both side to remember they need to be changed at the same time.